### ISSUE TITLE
Changes to protein responses

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/membrane-transport#207]
From discussion with @terracoda - 
- [x] Make grabbed/release responses "assertive".
- [x] Take the name off of the "grabbed" response.

**Comment by jessegreenberg** [comment]
I cannot see this problem with NVDA.

Unfortunately, my Mac with Big Sur 11.7.6 is not useful for testing this sim. All I hear is "Safari Busy" when I run with VoiceOver.

But I suspect that VoiceOver ignores the "Grabbed" alert and just speaks the focus alert when they happen so quickly.  Potential options:
1) We could delay the focus event somehow
```ts
    // Alert 'grabbed' before updating focus so that the 'grabbed' response is heard before the name of the protein.
    this.alert( this.grabReleaseUtterance, MembraneTransportFluent.a11y.transportProtein.grabbedResponseStringProperty );
    MembraneTransportSounds.transportProteinGrabbed();

    // Make sure that the selected index is set before the grabbedProperty, so that the focus is set correctly.
    window.setTimeout( () => {
      this.grabbedProperty.value = true;
    }, 1500 );
```

2) Try speaking the "grabbed" alert seconds in case that helps VoiceOver.
3) Try speaking the "grabbed" alert second, and with a delay, possibly using `alertStableDelay` - a built in Utterance option that lets the Utterance sit in the queue for a duration.

**Comment by samreid** [comment]
@jessegreenberg confirmed it still sounds great on NVDA, so we believe this issue is ready to close.
